### ISSUE TITLE
Playlist parser fails for unknown attributes in EXT-X-STREAM-INF

### ISSUE THREAD

**Issue by MTJack** [0xboobface/ctbrec#87]
<img width="360" alt="screen shot 2018-11-30 at 11 22 16 am" src="https://user-images.githubusercontent.com/34790527/49268503-61eb1900-f492-11e8-886d-90388c59f5cf.png">

When I opened a chaturbate live stream in VLC player, it showed me like this. And I am unable to record that live stream as well.

**Comment by 0xboobface** [comment]
Thanks for the feedback. Got a similar report already. This should be fixed with commit 2c716d3.

**Comment by 0xboobface** [comment]
I got the same error. It turns out, it is not enough to set the parser to lenient. Sometimes the playlist looks like the first listing. In the next moment it can look like the second listing. I have no idea why that happens. The problem is the NAME attribute, which is unknown to the parser.
```
#EXTM3U
#EXT-X-VERSION:4
#EXT-X-STREAM-INF:BANDWIDTH=488000,NAME="FPS:29.97",CODECS="avc1.42c015,mp4a.40.2",RESOLUTION=426x240
chunklist_w848253977_b448000_t64RlBTOjI5Ljk3.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1258000,NAME="FPS:29.97",CODECS="avc1.4d401f,mp4a.40.2",RESOLUTION=854x480
chunklist_w848253977_b1148000_t64RlBTOjI5Ljk3.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=3396000,NAME="FPS:29.97",CODECS="avc1.4d401f,mp4a.40.2",RESOLUTION=1280x720
chunklist_w848253977_b3096000_t64RlBTOjI5Ljk3.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=5628000,NAME="FPS:29.97",CODECS="avc1.640028,mp4a.40.2",RESOLUTION=1920x1080
chunklist_w848253977_b5128000_t64RlBTOjI5Ljk3.m3u8
```

```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-STREAM-INF:BANDWIDTH=488000,CODECS="avc1.66.21,mp4a.40.2",RESOLUTION=426x240
chunklist_w1885076366_b448000.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=1258000,CODECS="avc1.77.31,mp4a.40.2",RESOLUTION=854x480
chunklist_w1885076366_b1148000.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=3396000,CODECS="avc1.77.31,mp4a.40.2",RESOLUTION=1280x720
chunklist_w1885076366_b3096000.m3u8
#EXT-X-STREAM-INF:BANDWIDTH=5628000,CODECS="avc1.100.40,mp4a.40.2",RESOLUTION=1920x1080
chunklist_w1885076366_b5128000.m3u8
```
